docs/zh/latest/discovery/zookeeper.md

```diff
@@ -0,0 +1,143 @@
+---
+title: zookeeper
+---
```

Comment:
please add keywords and description. please refer: [installation ](https://github.com/apache/apisix/blob/master/docs/zh/latest/installation-guide.md)